reporting_app/templates/untabbed_datatables.html

```diff
@@ -20,4 +20,270 @@
     {{ datatable(table) }}
 {% endif %}
 
-{% endblock %}
+
+<script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
```

Comment:
This is not related to setting up datatables, so should be split off into its own template and .js script. There should then be a new 'pipeline_view' template that mixes in this and `untabbed_datatables.html`.